README.md

```diff
@@ -1 +1,3 @@
-# Analytics
\ No newline at end of file
+# Analytics
+
+link para el looker studio BQ 2.3: https://lookerstudio.google.com/s/j8pzo3OZxkg
```

Comment:
Hey can you update this to be in English? Thank you!

Reply:
Also with the file names!

Reply:
Also, to ensure consistency, rename the folder to BQ 2.3

Reply:
Hey @jdcastellanosb73 ! Can you change the folder name (add the space) and file names (to English) as well?